### PR TITLE
Fix tests code to prevent some runtime errors

### PR DESCRIPTION
**Pull request by Morlinest**
Trying to improve tests code as in #2376.

### CODE REVIEW

File: models/user_openid_test.go [hunk 1]
```diff
@@ -15,18 +15,20 @@ func TestGetUserOpenIDs(t *testing.T) {
 
 	oids, err := GetUserOpenIDs(int64(1))
 	if assert.NoError(t, err) {
-		assert.Len(t, oids, 2)
-		assert.Equal(t, oids[0].URI, "https://user1.domain1.tld/")
-		assert.False(t, oids[0].Show)
-		assert.Equal(t, oids[1].URI, "http://user1.domain2.tld/")
-		assert.True(t, oids[1].Show)
+		if assert.Len(t, oids, 2) {
```
Comment (ethantkoenig) [inline models/user_openid_test.go]: Maybe it'd be better to combine the two `if` statements to reduce nesting
```
if assert.NoError(t, err) && assert.Len(t, oids, 2) {
...
}
```

**Review by ethantkoenig — APPROVED** [review]
One minor suggestion

**Comment by ethantkoenig** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

File: models/user_openid_test.go (hunk 1 above)
Comment (Morlinest) [reply models/user_openid_test.go]: You're right

**Comment by lafriks** [comment]
Make LG-TM work

**Pull request merged**